### ISSUE TITLE
System.Resources.Reader/Writer packages needs Xamarin support

### ISSUE THREAD

**Issue by akoeplinger** [dotnet/corefx#27995]
We've added facades for System.Resources.Reader.dll and System.Resources.Writer.dll inbox so we'd like to add the placeholders for the usual Xamarin platforms to those packages.

**Comment by weshaggard** [comment]
We no longer build and ship those packages live, so it would need to happen in servicing event if we really need/want this. Do we believe this is worth servicing for?

**Comment by marek-safar** [comment]
@weshaggard I am curious .net 4.7.1 ships this package as inbox how is it going to be picked up?

**Comment by weshaggard** [comment]
That is a great question it looks like these library packages only contain a netstandard1.0 or 1.3 lib which means that if they are consumed on .NET 4.7.1 they will be an independent duplicated library and will not unify with desktop (bug in our packages :(). We did we put these shims inbox on .NET 4.7.1 so any assembly references to these will unify but the package references will not.

I chatted with @ericstj and we don't believe these packages are all that interesting for folks to use but if they do end up being pulled in folks could add a ExcludeAssets=all on the PackageReference to workaround. 

I assume you guys found this from the work putting the facades inbox and not via customer reports correct? If so I don't think this is going to meet our servicing bar to fix at this point, and we should just keep the workaround in our back-pocket for folks if they hit this and if we get enough reports then we can service the packages.
